tests/unit/challenges/test_aws_utils.py

```diff
@@ -0,0 +1,201 @@
+import boto3
+import mock
+import os
+
+from datetime import timedelta
+from http import HTTPStatus
+from moto import mock_ecs
+
+from allauth.account.models import EmailAddress
+from django.contrib.auth.models import User
+from django.utils import timezone
+
+from rest_framework.test import APITestCase, APIClient
+
+import challenges.aws_utils as aws_utils
+from challenges.models import Challenge
```

Comment:
@nikochiko please reorder all imports lexicographically. Another thing all `from` imports should come at last and all the direct `import` statements should be at start. Your lines 5-7 are correct but all from imports should be considered together to order lexicographically

Reply:
Done! 👍 Please see if it is okay now.